### PR TITLE
prisma: Fix autocompletion adding word character

### LINKED ISSUE #13662: Prisma autocomplete with "@" symbol works incorrect
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When I try to use autocomplete with "@" symbol (ex. @map, @relation, @db, etc.) I got extra "@" symbols

### Environment

Zed: v0.141.3 (Zed)
OS: macOS 15.0.0
Memory: 8 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

https://github.com/zed-industries/zed/assets/91194878/5ee7609c-14ac-4ad7-85ad-99c9b7bd3c31

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

### PR DESCRIPTION
**Pull request by kriskw1999**
Fixes https://github.com/zed-industries/zed/issues/13662.

Release Notes:

- N/A

### CODE REVIEW

File: extensions/prisma/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zed_prisma"
-version = "0.0.2"
+version = "0.0.3"
```
Comment (maxdeviant) [inline extensions/prisma/Cargo.toml:3]: Let's remove the version bump and just do the fix in this PR.

We can bump the extension version afterwards.

File: extensions/prisma/languages/prisma/config.toml [hunk 2]
```diff
@@ -8,3 +8,7 @@ brackets = [
     { start = "(", end = ")", close = true, newline = true }
 ]
 tab_size = 2
+word_characters = ["@"]
+
+[overrides.string]
+word_characters = ["@"]
```
Comment (maxdeviant) [inline extensions/prisma/languages/prisma/config.toml:14]: This file should have a trailing newline:

```suggestion
word_characters = ["@"]

```

**Comment by kriskw1999** [comment]
@maxdeviant completed both the points, thanks!

**Review by maxdeviant — APPROVED** [review]
Thanks!

**Comment by nothugofsea** [comment]
Thanks!

**Pull request merged**